### ISSUE TITLE
TypeError: _set_position() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by saildata** [pvcraven/arcade#324]
### Actual behavior:
```
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/examples/asteroid_smasher.py", line 394, in <module>
    main()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/examples/asteroid_smasher.py", line 390, in main
    arcade.run()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/window_commands.py", line 245, in run
    pyglet.app.run()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/pyglet/app/__init__.py", line 142, in run
    event_loop.run()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/pyglet/app/base.py", line 175, in run
    self._run()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/pyglet/app/base.py", line 187, in _run
    timeout = self.idle()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/pyglet/app/base.py", line 308, in idle
    redraw_all = self.clock.call_scheduled_functions(dt)
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/pyglet/clock.py", line 314, in call_scheduled_functions
    item.func(now - item.last_ts, *item.args, **item.kwargs)
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/examples/asteroid_smasher.py", line 355, in update
    self.all_sprites_list.update()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/sprite_list.py", line 299, in update
    sprite.update()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/examples/asteroid_smasher.py", line 103, in update
    super().update()
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/sprite.py", line 547, in update
    self.set_position(self.center_x + self.change_x, self.center_y + self.change_y)
  File "/home/srw/projects/pyarcade_lab/venv/lib/python3.7/site-packages/arcade/sprite.py", line 202, in set_position
    self._set_position(self, (center_x, center_y))
TypeError: _set_position() takes 2 positional arguments but 3 were given
```
### Expected behavior: 
Runs without error

### Steps to reproduce/example code:
git checkout f41a10695d7689a37a88b5b35161cd6351063fd1
python -m arcade.examples.asteroid_smasher 

### What works:
git checkout cf77862dcddee555bf68d1adaeb82e780c092cb9
python -m arcade.examples.asteroid_smasher

**Comment by pvcraven** [comment]
Thanks for the report.

**Comment by pvcraven** [comment]
Closed with 2.0 release.
